sw/cheri/README.md

```diff
@@ -0,0 +1,51 @@
+# CHERIoT programs for the Sonata system
+
+See the [getting started guide](../../doc/guide/getting-started.md) for how to install dependancies.
+
+## Building
+
+To build, run the following from the root of the directory.
+
+```
+cmake -G Ninja -B build_cheri -S sw/cheri
```

Comment:
I would avoid `-G Ninja` in the doc to avoid adding extra dependency. Most people have `make` installed but not necessary ninja.